src/test/java/pl/neumann/demoapp/api/ProductEndPointTest.java

```diff
@@ -0,0 +1,96 @@
+package pl.neumann.demoapp.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import pl.neumann.demoapp.DemoappApplicationTests;
+import pl.neumann.demoapp.domain.ProductFacade;
+import pl.neumann.demoapp.domain.ProductRequestDto;
+import pl.neumann.demoapp.domain.ProductResponseDto;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class ProductEndPointTest extends DemoappApplicationTests {
```

Comment:
Proponuję dodać strukturę _given...when...then_, wpłynie to pozytywnie na czytelność :)